### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by tintunaung1988** [jup-ag/token-list#738] [General]
hello

**Comment by Fury211** [comment]
hi

**Comment by DOGECOIN87** [comment]
Yo

**Comment by MONTPBM** [comment]
WTH IS GOING ON WITH MY MONEY AND TOKENS????!!!! YOU OR JUP IS SCAMMING US RIGHT NOW!

**Reply by mellowmeli** [reply]
This account/token is completely frozen! Looks like a scam!

**Comment by byamartins** [comment]
Hi!

I was opening a position and my solana is gone but I cannot see my position anywhere? Can someone check?
